understanding/20/images-of-text-no-exception.html

```diff
@@ -9,6 +9,15 @@
    <h1>Understanding Images of Text (No Exception)</h1>
    
    
+   <section id="brief">
+      <h2>In brief</h2>
+      <dl>
+         <dt>Objective</dt><dd>Only use text that can be adapted by users</dd>
+         <dt>Author task</dt><dd>Avoid pictures of text except where essential</dd>
```

Comment:
```suggestion
         <dt>Author task</dt><dd>Do not use pictures of text except where unavoidable</dd>
```

Reply:
Based on a suggestion from Alastair, changing this to
```suggestion
         <dt>Author task</dt><dd>Do not use pictures of text unless there is no other way to present information</dd>
```